src/mappings/primitives.ts

```diff
@@ -163,6 +165,8 @@ async function _handleBlock(block: CosmosBlock): Promise<void> {
   });
 
   await blockEntity.save();
+
+  await _handleSupply(block);
```

Comment:
#PUC why we're doing this here for now and that it's temporary (i.e. will not be block-driven once claim settlement event is updated).